components/style/values/specified/font.rs

```diff
@@ -13,7 +13,7 @@ use properties::longhands::system_font::SystemFont;
 use std::fmt;
 use style_traits::{ToCss, StyleParseErrorKind, ParseError};
 use values::computed::{font as computed, Context, Length, NonNegativeLength, ToComputedValue};
-use values::specified::{AllowQuirks, LengthOrPercentage, NoCalcLength};
+use values::specified::{AllowQuirks, LengthOrPercentage, NoCalcLength, Number as SpecifiedNumber};
```

Comment:
Why `SpecifiedNumber`? Let's just use `Number`, since this is the `specified` module.

Reply:
I originally think it will become `Number(Number)` in `FontSizeAdjust` enum if using `Number` but I think you're right. I've updated it. Using `SpecifiedNumber` looks redundant.